examples/3d/3d_gizmos.rs

```diff
@@ -191,6 +191,14 @@ fn draw_example_collection(
         .circle_segments(64);
 
     gizmos.arrow(Vec3::ZERO, Vec3::ONE * 1.5, Color::YELLOW);
+
+    {
+        // We can create more complex arrows using the arrow builder.
+        let mut arrow_builder =
+            gizmos.arrow(Vec3::ZERO, Vec3::from_array([-1.5, 1.5, 1.5]), Color::GREEN);
+        arrow_builder.with_double_end();
+        arrow_builder.with_tip_length(0.5);
+    };
```

Comment:
```suggestion
    // You can create more complex arrows using the arrow builder.
    let mut arrow_builder =
        gizmos.arrow(Vec3::ZERO, Vec3::from_array([-1.5, 1.5, 1.5]), Color::GREEN);
    arrow_builder.with_double_end();
    arrow_builder.with_tip_length(0.5);
```
1. Remove the unecessary brackets;
2. NIT: Change the comment to match the existing comments narrative;

Reply:
Also I noticed by default we can't see the "start" tip of the arrow, since its inside the cube. To showcase the double-ended arrow, it would be better if the arrow as moved to have its both ends visible right up front.